yb-voyager/src/queryparser/unsupported_constructs_test.go

```diff
@@ -0,0 +1,231 @@
+package queryparser
+
+import (
+	"testing"
+
+	pg_query "github.com/pganalyze/pg_query_go/v5"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+// TestFuncCallDetector tests the Advisory Lock Detector.
+func TestFuncCallDetector(t *testing.T) {
```

Comment:
nice tests! How about one with all three present, just to ensure that multiple constructs are returned.

Reply:
done